test/framework/operator.go

```diff
@@ -127,7 +93,7 @@ var (
 // DeployNamespacedKogitoOperatorUsingOlm Deploy Kogito Operator watching for specified namespace using OLM, return all objects created for deployment
 // Will be deployed in the same namespace as it will watch for
 func DeployNamespacedKogitoOperatorUsingOlm(deploymentNamespace string) error {
```

Comment:
Do we still need this?

Reply:
No, we don't, will delete it. Thanks for noticing.